src/nvim/msgpack_rpc/channel.c

```diff
@@ -721,7 +720,12 @@ static void complete_call(msgpack_object *obj, Channel *channel)
   }
 }
 
-static void call_set_error(Channel *channel, char *msg)
+/// Set the channel's call stack to indicate error state.
+/// Closes and frees the channel.
+///
+/// @param channel The Channel to close.
```

Comment:
I generally encourage people to avoid tautological param notes like this. Leaving it blank would be fine, since the other param has actually useful documentation